src/components/templates/AnimateMap/game/constants/AssetConstants.ts

```diff
@@ -37,8 +37,8 @@ import HAT_17 from "assets/images/AnimateMap/avatarHats/cap_22.png";
 import HAT_18 from "assets/images/AnimateMap/avatarHats/cap_23.png";
 import BARREL_0 from "assets/images/AnimateMap/barrels/barrel.png";
 import FIREBARELL_CAM_ICON from "assets/images/AnimateMap/barrels/camIcon.png";
-import FIREBARELL_HALO from "assets/images/AnimateMap/barrels/halo.png";
-import FIREBARELL_HALO_GREEN from "assets/images/AnimateMap/barrels/haloGreen.png";
```

Comment:
do you remove the images that are no longer used in the project? or they are still needed?